### DISCUSSION TITLE
Disable built-in authentication option

### DISCUSSION THREAD

**Discussion by robflate** [autobrr/autobrr#1907] [Ideas]
Thanks for adding OIDC authentication. The docs say;

> If you have an existing user in the database, both OIDC and built-in authentication will be available

Can I request an option to disable built-in authentication even if a user exists in the database? Preferably this can be set with an environment variable in Docker.

It is useful to be able to login with built-in authentication just in case something goes wrong with the OIDC provider but for the majority of the time I don't want to expose the built-in authentication. Thanks.

**Comment by s0up4200** [comment]
Hi! That shouldn't be a problem. I'll reference this post once I have a PR up.

**Comment by s0up4200** [comment]
PR https://github.com/autobrr/autobrr/pull/1908

**Comment by intriguedlife** [comment]
Hi,

I would like to automatically start the OIDC login flow without having to click on the “OpenID Connect” button.
Is there a url that directly starts the OIDC login flow?
It looked to me like there was javascript involved from looking at the login page.
